### PR TITLE
TX2 ARM-Based Docker Build

### PR DESCRIPTION
**Pull request by Vuwij**

### CODE REVIEW

File: .github/workflows/docker_image_arm.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          password: ${{ secrets.DOCKER_TOKEN }}
+      - name: Set up Docker Buildx      
+        uses: crazy-max/ghaction-docker-buildx@v1      
+        with:        
+          version: latest    
+      - name: Build the Docker image
+        run: |
+          docker pull utrarobosoccer/soccerbot:arm64 || echo "Starting from scratch"
+          docker buildx build . --file Dockerfile --platform linux/arm64 --tag utrarobosoccer/soccerbot:arm64 --cache-from utrarobosoccer/soccerbot:arm64 --build-arg BUILDKIT_INLINE_CACHE=1 --build-arg BASE_IMAGE=arm64v8/ros:noetic-robot
+          # docker buildx build . --file Dockerfile --platform linux/arm64 --tag utrarobosoccer/soccerbot:arm64 --cache-from utrarobosoccer/soccerbot:arm64 --build-arg BUILDKIT_INLINE_CACHE=1 --build-arg BASE_IMAGE=dustynv/ros:noetic-ros-base-l4t-r32.7.1
```
Comment (Vuwij) [inline .github/workflows/docker_image_arm.yml:42]: delete comment

File: Dockerfile [hunk 2]
```diff
@@ -39,7 +42,7 @@ RUN apt-fast install -y \
     apt-utils \
     software-properties-common \
     sudo \
-    ros-noetic-robot-state-publisher \
```
Comment (Vuwij) [inline Dockerfile:42]: can you try adding these back

File: Dockerfile [hunk 3]
```diff
@@ -49,11 +52,17 @@ RUN apt-fast install -y \
     libxcb-render-util0 \
     libxcb-randr0 \
     libxcb-keysyms1 \
-    libxcb-xinerama0
+    libxcb-xinerama0 \
+    python3.8-dev \
+    python3-pyqt5
 RUN curl -sSL https://get.docker.com/ | sh
```
Comment (Vuwij) [inline Dockerfile:58]: no longer needed

File: docker-compose.yaml [hunk 4]
```diff
@@ -26,6 +26,7 @@ x-soccerbot: &soccerbot
     context: .
     target: builder
     args:
+      - BASE_IMAGE=dustynv/ros:noetic-ros-base-l4t-r32.7.1
```
Comment (Vuwij) [inline docker-compose.yaml:29]: remove this

File: requirements.txt [hunk 5]
```diff
@@ -14,7 +13,7 @@ shapely
 vispy
 pytest
 protobuf>=3.19.4
-tensorflow
+#tensorflow
```
Comment (Vuwij) [inline requirements.txt:16]: try adding all back

**Review by Shah-Rajab — APPROVED** [review]

**Pull request merged**